src/gui/editorwidgets/core/qgswidgetwrapper.h

```diff
@@ -180,6 +218,11 @@ class GUI_EXPORT QgsWidgetWrapper : public QObject
     QWidget *mParent = nullptr;
     QgsVectorLayer *mLayer = nullptr;
     bool mInitialized;
+
+    //! Property definitions
+    static QgsPropertiesDefinition sPropertyDefinitions;
```

Comment:
Can't we put that into `propertyDefinitions ()` like I did? For less header pollution?